packages/integrations/sagas/gtmSaga.js

```diff
@@ -0,0 +1,25 @@
+import {
+  takeLatest,
+} from 'redux-saga/effects';
+import {
+  RECEIVE_PICO_VERIFICATION_FAILURE,
+  RECEIVE_PICO_PLAN_UPGRADE,
+  REQUIRE_UPGRADE_FOR_CORAL_SSO,
+} from '../actions/types';
+
+// The GTM saga.
+export default [
+  takeLatest(REQUIRE_UPGRADE_FOR_CORAL_SSO, sendPicoEvent),
```

Comment:
`redux-saga`'s take patterns can take multiple effects for a single pattern. This could be simplified to:
```suggestion
  takeLatest(
    [
      REQUIRE_UPGRADE_FOR_CORAL_SSO,
      RECEIVE_PICO_VERIFICATION_FAILURE,
      RECEIVE_PICO_PLAN_UPGRADE,
    ], sendPicoEvent
  ),
```